### PR TITLE
Wrap ownerOf function in try/catch:

### PR DESCRIPTION
**Pull request by atbe**

### CODE REVIEW

File: src/modules/nft.ts [hunk 1]
```diff
@@ -73,8 +73,19 @@ export class NFTModule extends ModuleWithRoles<NFT> {
...
+   * @param tokenId - ID of the NFT to get the owner of
+   * @returns the owner of the token, or an empty string if the
+   * token has been burned
+   */
   public async ownerOf(tokenId: string): Promise<string> {
-    return await this.readOnlyContract.ownerOf(tokenId);
+    try {
+      return await this.readOnlyContract.ownerOf(tokenId);
+    } catch (e) {
+      return "";
```
Comment (jakeloo) [inline src/modules/nft.ts:87]: do `ethers.constants.AddressZero` for whenever we return addresses.

File: src/modules/nft.ts [hunk 2]
```diff
@@ -73,8 +73,19 @@ export class NFTModule extends ModuleWithRoles<NFT> {
...
+   *
+   * @param tokenId - ID of the NFT to get the owner of
+   * @returns the owner of the token, or an empty string if the
+   * token has been burned
+   */
   public async ownerOf(tokenId: string): Promise<string> {
-    return await this.readOnlyContract.ownerOf(tokenId);
+    try {
+      return await this.readOnlyContract.ownerOf(tokenId);
+    } catch (e) {
```
Comment (atbe) [inline src/modules/nft.ts:86]: I should call out that this is a pretty general error handler here and that we may signal that a token is burned even if the call failed for a different reason.

**Review by jakeloo — CHANGES_REQUESTED** [review]

File: src/modules/nft.ts (hunk 1 above)
Comment (atbe) [reply src/modules/nft.ts]: Good catch, fixed!

**Comment by atbe** [comment]
Please squash merge!

**Comment by jakeloo** [comment]
feel free to merge it and make a patch build.

@jnsdls the process of making the sdk build is:
1. merge into main
2. yarn build
3. np

right?

**Comment by jnsdls** [comment]
> feel free to merge it and make a patch build.
> 
> @jnsdls the process of making the sdk build is:
> 
> 1. merge into main
> 2. yarn build
> 3. np
> 
> right?

yes

**Pull request merged**